### PR TITLE
Disable Nagle algorithm

### PR DESCRIPTION
**Pull request by patuwwy**
**What?**  
Disable Nagle algorithm

**Why?**  
checking performance

### CODE REVIEW

**Comment by MichalCz** [comment]
Patryk, can we merge this?

**Review by alicja-gruzdz — APPROVED** [review]

**Comment by patuwwy** [comment]
@MichalCz now we can

**Comment by MichalCz** [comment]
I'd like to run this before. @alicja-gruzdz @daro1337 perhaps we should do a patch release before releasing this? So 0.28.4 before this in 0.28.5

**Review by ErykSol — APPROVED** [review]

File: packages/verser/src/lib/verser.ts [hunk 1]
```diff
@@ -23,9 +24,12 @@ export class Verser extends TypedEmitter<Events> {
         super();
         this.server = server;
 
-        this.server.on("connect", (req, socket) => {
+        this.server.on("connect", (req, socket: Socket) => {
             this.logger.info("New connection:", req.url);
 
+            socket.setTimeout(0);
```
Comment (ErykSol) [inline packages/verser/src/lib/verser.ts:30]: As I understand timeout=0 is just for testing, not as permanent solution?

Comment (patuwwy) [reply packages/verser/src/lib/verser.ts:30]: permanent, https://nodejs.org/api/net.html#socketsettimeouttimeout-callback

Comment (ErykSol) [reply packages/verser/src/lib/verser.ts:30]: Ah now I see, thx for claryfying :)

**Review by alicja-gruzdz — APPROVED** [review]

**Comment by patuwwy** [comment]
can we merge this one?

**Comment by alicja-gruzdz** [comment]
> can we merge this one?

In my opinion yes since it showed green in CPM.

**Pull request merged**